apps/storybook/src/Contexts.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta title="Contexts" />
+
+## Contexts
+
+Children of `VisCanvas` can access two React contexts: `AxisSystemContext` and `CameraContext`.
+
+### AxisSystemContext
+
+`AxisSystemContext` includes helpful utilities to convert vectors from data space to world space and back.
+It exposes the size of the visualization, as well as the axis configs passed to `VisCanvas` for convenience.
+It also provides an API to implement custom interactions.
+
+This context re-renders **whenever the size of the canvas changes**.
+
+```ts
+useAxisSystemContext(): AxisSystemParams
+
+const { visSize, dataToWorld, worldToData } = useAxisSystemContext();
```

Comment:
I no longer list all the available properties. The point is to give an example of use. The full list is already in the table below.